test/content_test.exs

```diff
@@ -1,26 +1,14 @@
 defmodule ContentTest do
   use ExUnit.Case
   use Plug.Test
-  # @opts AuthPlug.init(%{auth_url: "https://dwylauth.herokuapp.com"})
 
-
-  def dummy(conn, _params) do
-    # IO.inspect(conn.halted, label: "conn.halted")
-    # IO.inspect(conn, label: "conn:9")
-    conn
-    |> assign(:dummy, "hello")
-  end
+  def dummy(conn, _params), do: assign(conn, :dummy, "hello")
```

Comment:
guessing this is `mix format` tidying up?

Reply:
This one is me, but I don't mind to put it back to `do end` format

Reply:
Nah, just curious.